app/lib/rocketchat/rocketchat.js

```diff
@@ -683,7 +683,7 @@ const RocketChat = {
 	readMessages,
 	resendMessage,
 
-	async localSearch({ text, filterUsers = true, filterRooms = true }) {
+	async localSearch({ text, filterUsers = true, filterRooms = true, searchView = false }) {
```

Comment:
We shouldn't have to add this extra param.

Reply:
Added this just to check if we want the data for the search view . Do we have any other workaround?